cypress/integration/page.spec.js

```diff
@@ -1078,16 +1162,22 @@
 
       postDetails.comments().should('have.length', 6);
 
-      postDetails.comments().eq(5).byDataCy('CommentAuthor')
+      postDetails
+        .comments()
+        .eq(5)
+        .byDataCy('CommentAuthor')
         .should('have.text', 'Some name')
         .and('have.attr', 'href', 'mailto:some@email.com');
 
-      postDetails.comments().eq(5).byDataCy('CommentBody')
+      postDetails
+        .comments()
+        .eq(5)
+        .byDataCy('CommentBody')
         .should('have.text', 'Some comment body');
```

Comment:
Ensure that the `cy.get` and `its` statements are properly chained and terminated with semicolons. This ensures proper execution of the assertions.